Documentation/Blazorise.Docs/Components/ComponentApiDocs.razor

```diff
@@ -0,0 +1,48 @@
+﻿@using Blazorise.Generator.Features.ApiDocsDtos
+<DocsPageSubtitle>
+    API
+</DocsPageSubtitle>
+
+@{
+    IReadOnlyList<ApiDocsForComponent> apiDocsForComponents = ComponentTypes.Select(x => ComponentsApiDocsSource.Instance.Components.GetValueOrDefault(x)).Where(x => x is not null).ToList();
+    var hasComponentTypes = ComponentTypes.Any();
+    var multipleComponentTypes = ComponentTypes.Count > 1;
+    var hasMethods = apiDocsForComponents.Any(x => x.Methods.Any());
```

Comment:
Could all these calculations be done on the OnInit method just once?
It would be a little bit more consistent and tidy? We would also be sure that this won't be recalculated needlessly?
Would you agree?